support-frontend/assets/helpers/subscriptions.js

```diff
@@ -184,7 +184,7 @@ const getNewsstandSaving = (subscriptionMonthlyCost: number, newsstandWeeklyCost
   fixDecimals(getMonthlyNewsStandPrice(newsstandWeeklyCost) - subscriptionMonthlyCost);
 
 const getNewsstandSavingPercentage = (subscriptionMonthlyCost: number, newsstandWeeklyCost: number) =>
-  Math.round(100 - ((subscriptionMonthlyCost / getMonthlyNewsStandPrice(newsstandWeeklyCost)) * 100));
+  Math.floor(100 - ((subscriptionMonthlyCost / getMonthlyNewsStandPrice(newsstandWeeklyCost)) * 100));
```

Comment:
I'm not sure why I understand why this is necessary other than to make it match the numbers marketing have used.